core/src/databases/table.rs

```diff
@@ -702,24 +702,40 @@ impl Row {
                 Value::Bool(bool_val)
             } else {
                 // Various datetime formats
-                let dt: Option<DateTime<Utc>> = [
+                let mut dt: Option<DateTime<Utc>> = [
                     // RFC3339
                     DateTime::parse_from_rfc3339(trimmed).map(|dt| dt.into()),
                     // RFC2822
                     DateTime::parse_from_rfc2822(trimmed).map(|dt| dt.into()),
-                    // Google Spreadsheet format
-                    DateTime::parse_from_str(trimmed, "%d-%b-%Y").map(|dt| dt.into()),
                     // SQL
                     DateTime::parse_from_str(trimmed, "%Y-%m-%d %H:%M:%S").map(|dt| dt.into()),
                     // HTTP date
                     DateTime::parse_from_str(trimmed, "%a, %d %b %Y %H:%M:%S GMT")
                         .map(|dt| dt.into()),
+                    // Google Spreadsheet format
+                    NaiveDate::parse_from_str(trimmed, "%d-%b-%Y").map(|d| {
```

Comment:
TIL cc @aubin-tchoi DateTime can't parse non fully functional dates.

Reply:
By fully functional I mean fully defined date and time. If the parse string is just a date it will error